interop-cats/shared/src/test/scala/zio/interop/GenIOInteropCats.scala

```diff
@@ -48,6 +48,9 @@ trait GenIOInteropCats {
   def genIO[E: Arbitrary, A: Arbitrary]: Gen[IO[E, A]] =
     Gen.oneOf(genSuccess[E, A], genFailure[E, A])
 
+  def genUIO[A: Arbitrary]: Gen[UIO[A]] =
```

Comment:
`genOfRace` below needs to be changed to:

```scala
  private def genOfRace[E, A](io: IO[E, A]): Gen[IO[E, A]] =
    Gen.const(io.raceFirst(ZIO.never))
```

The current implementation in terms of `race` means we wait for the second fiber if the first fiber fails, which means if the first fiber fails we suspend indefinitely.